profiles/sabhisharma-ise.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "![image](https://github.com/EddieHubCommunity/awesome-github-profiles/assets/100849696/7a94a885-6d68-480a-b81c-e1860e63909c)",

```

Only the URL must be provided, and not in Markdown format.
```suggestion
    "image": "https://github.com/EddieHubCommunity/awesome-github-profiles/assets/100849696/7a94a885-6d68-480a-b81c-e1860e63909c",
```